templates/vars-30.epp

```diff
@@ -0,0 +1,80 @@
+# easy-rsa parameter settings
```

Comment:
does this actually need to be a template? Can't we use the files distributed by the operatingsystems?

Reply:
For me, template usage is still relevant because the file `vars` need to be customized for each `openvpn` tunnel.

If you are thinking about something else. Please tell me more.

Reply:
ah okay. Than it makes sense.